charts/dex.yaml

```diff
@@ -0,0 +1,98 @@
+image: '{%- endraw -%}{{ build_image_name(\"dex\", False) }}{%- raw -%}'
```

Comment:
(unrelated to this line, cannot comment on the commit message)

This commit message states: "add `dex` deployment configuration", but it actually also includes the rendered chart. To me, this commit and the previous one should be squashed.

Reply:
Also, remove the note about `Secret`  not being rendered properly, as this is fixed in the same PR (just re-order commit to fix it before rendering, and squash all commits including the rendered chart).